static/js/components/GcnSelectionForm.jsx

```diff
@@ -62,14 +62,18 @@ const GcnSelectionForm = ({ gcnEvent }) => {
     "instrument",
     "observations",
   ];
+  const displayOptionsDefault = displayOptions.reduce(
```

Comment:
This is cute, but not very readable. Perhaps consider:

```
Object.fromEntries(displayOptions.map(x => [x, false]))
```

Reply:
Yup added.